### PR TITLE
Feature/vr 69

### PR DESCRIPTION
**Pull request by n3op2**
# What

Basic layout elements https://digicatapult.atlassian.net/browse/VR-69:
- SideBar - main nav
- Content 
    - Header - rendering some current info e.g. path, form titles and etc
    - Main - for rendering data and user views
    
## screenshots
Screenshot 2024-05-10 at 08 30 08
Screenshot 2024-05-10 at 08 30 00

### CODE REVIEW

File: public/styles/main.css [hunk 1]
```diff
@@ -3,13 +3,62 @@
...
-  justify-items: center;
   font-family: system-ui, sans-serif;
-  font-weight: 500;
-  font-size: 10rem;
+}
+
+body.flex-page {
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
```
Comment (mattdean-digicatapult) [inline public/styles/main.css:15]: you probably don't need `width` as it's a block level element. It should just fill available space. Can you try removing any width 100% or 100vw that are applied to divs and check I'm right. This is somewhat important as it reduces the odds of random scrollbars appearing

File: src/views/common.tsx [hunk 2]
```diff
@@ -9,9 +15,16 @@ export const Page = (props: Html.PropsWithChildren<{ title: string }>): JSX.Elem
         <script src="lib/htmx.org/ext/json-enc.js"></script>
         <link rel="icon" type="image/ico" sizes="48x48" href="/public/images/favicon.ico" />
         <link rel="stylesheet" type="text/css" href="/public/styles/main.css" />
+        <link rel="stylesheet" type="text/css" href="/public/styles/semantic.css" />
```
Comment (mattdean-digicatapult) [inline src/views/common.tsx:18]: does this exist?

Comment (n3op2) [reply src/views/common.tsx]: nope, tested something earlier another css for some basics e.g. layout

File: public/styles/main.css (hunk 1 above)
Comment (n3op2) [reply public/styles/main.css]: this should be updated, thanks for the flag up

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**